### DISCUSSION TITLE
Will more languages be supported?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [trpc/trpc#3593] [Q&A] [answered]
Or is it going to be Typescript exclusive?

**Comment by juliusmarminge** [comment]
trpc is not tied to node or any runtime. We have examples for both Node, Deno and Bun.

It's also not bound to http.

**Reply by shinebayar-g** [reply]
Sorry, meant to say Typescript.

**Reply by KATT** [reply, marked as answer]
If you want support for other languages, check out [trpc-openapi](https://github.com/jlalmes/trpc-openapi)

**Comment by Nick-Lucas** [comment]
Proper OpenAPI support is coming soon and this should enable the use of tRPC APIs across other languages, please help us out by trying the package, following the thread, and sharing your feedback and requests!

https://github.com/trpc/trpc/discussions/7239

Going to close this to consolidate future discussion over there
